tools/backups2datalad/manager.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
+
+from dataclasses import InitVar, dataclass, field, replace
+from typing import Any, Optional
+
+from anyio.abc import AsyncResource
+from ghrepo import GHRepo
+import httpx
+from humanize import naturalsize
+
+from .adandi import RemoteDandiset
+from .adataset import DatasetStats
+from .aioutil import arequest
+from .config import BackupConfig
+from .consts import USER_AGENT
+from .logging import PrefixedLogger, log
+from .util import quantify
+
+
+@dataclass
+class Manager(AsyncResource):
```

Comment:
Could you please add a docstring to describe the purpose of such a manager (too generic name to discern its purpose)

Reply:
Done: 041d245.